SLAM_node:
  AUV_mass: 33.5          # Set the AUV mass value here
  AUV_added_mass: 3.35     # Set the AUV added mass value here
  I_matrix:                # Set the inertial matrix values here for +ve motion directions
    - [364.7894, 0.0  , 0.0  ,  0.0]
    - [0.0  , 265.3494, 0.0  ,  0.0]
    - [0.0  , 0.0  , 104.8941,  0.0]
    - [0.0  , 0.0  , 0.0  ,  7.5746]
  Tau_g:  # Set the gravitational torque vector here
    - [0.0] 
    - [0.0] 
    - [9.81]
    - [0.0]
  Tau_b:  # Set the buoyancy torque vector here
    - [0.0] 
    - [0.0] 
    - [-9.81]
    - [0.0]
  Drag_co:                 # Set the drag terms values here for +ve motion directions
    - [290.2090, 0.0, 0.0, 0.0]
    - [0.0, 324.7808, 0.0, 0.0]
    - [0.0, 0.0, 362.5386, 0.0]
    - [0.0, 0.0, 0.0, 16.49770]
  Lift_co:                 # Set the lift coefficient values here
    - [0.0, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0]
  R:                       # Set the motion model noise matrix here
    - [0.0001, 0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0  ]
    - [0.0  , 0.0001, 0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0  ]
    - [0.0  , 0.0  , 0.0001, 0.0  , 0.0  , 0.0  , 0.0  , 0.0  ]
    - [0.0  , 0.0  , 0.0  , 0.0001, 0.0  , 0.0  , 0.0  , 0.0  ]
    - [0.0  , 0.0  , 0.0  , 0.0  , 0.0001, 0.0  , 0.0  , 0.0  ]
    - [0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0001, 0.0  , 0.0  ]
    - [0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0001, 0.0  ]
    - [0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0  , 0.0001]
  C:                       # Set the output matrix "C" here
    - [1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    - [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0]
    - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
  ORB_sigma: 0.1           # Set the ORB variance value here
